dsa/CustomLinkedList.java

```diff
@@ -0,0 +1,275 @@
+import java.util.Scanner;
+
+public class CustomLinkedList {
+
+  public static void main(String[] args) {
+
+    try {
+      Scanner sc = new Scanner(System.in);
+      LinkedList ll = new LinkedList(); //ll- linkedlist
+
+      while (true) {
+        System.out.print("Your linked list: ");
+        ll.display();
+
+        System.out.println("Enter 1 to add at last");
+        System.out.println("Enter 2 to add at First");
+        System.out.println("Enter 3 to add at a position");
+        System.out.println("Enter 4 to remove from last");
+        System.out.println("Enter 5 to remove from First");
+        System.out.println("Enter 6 to remove at a position");
+        System.out.println("Enter 7 to get from last");
+        System.out.println("Enter 8 to get from First");
+        System.out.println("Enter 9 to get at a position");
+        System.out.println("Enter 10 to reverse the list");
+        System.out.println("Enter 11 to get the mid node");
+
+        int option = sc.nextInt();
+
+        if (option == 1) {
```

Comment:
Using switch case instead of else if will be better

Reply:
Done sir.